TOOLS/ce/src/main/scala/org.opalj.ce/ConfigEntry.scala

```diff
@@ -0,0 +1,87 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj
+package ce
+
+import org.apache.commons.text.StringEscapeUtils
+
+/**
+ * Stores a value inside the structure of the configNode.
+ *
+ * @param value is the value stored in the entry.
+ * @param comment are all the comments associated with the value.
+ */
+case class ConfigEntry(value: String, var comment: DocumentationComment) extends ConfigNode {
+    /**
+     * Formats the entry into HTML code.
+     * @param label required if the Entry is part of an object (Writes the key of the K,V Map there instead). Overrides the label property of the Comment object.
+     * @param HTMLHeadline accepts the HTML syntax of the Headline of the value. Can contain $ label and $ brief flags for filling with content.
+     * @param HTMLContent accepts the HTML syntax of the content frame for the value. Must contains a $ content flag for correct rendering.
+     * @param HTMLStringBuilder accepts a StringBuilder. The method adds the HTML String to this StringBuilder.
+     * @param sorted actually does nothing in this method, but is required for the whole structure.
+     * @param maximumHeadlinePreviewLength accepts an integer that determines the maximum amount of characters that the fallback brief preview can contain.
+     */
+    override def toHTML(
```

Comment:
Most of this method is identical or similar in all three classes (ConfigEntry, ConfigObject, ConfigList). Consider extracting the common parts.